csharp/Ice/CustomError/slice/Greeter.ice

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) ZeroC, Inc.
+
+#pragma once
+
+module VisitorCenter
+{
+    /// The errors returned by the Greeter.
+    enum GreeterError { EmptyName, NameTooLong, Away };
+
+    /// Represents an exception.
+    exception GreeterException {
+        /// The reason for the exception.
+        string reason;
+
+        //The ID for the exception, given as a GreeterError enum.
```

Comment:
Always `///` followed by a space.

And this error is not an ID, but an error code.